nft_ingester/src/raydium_price_fetcher.rs

```diff
@@ -36,6 +36,43 @@ impl RaydiumTokenPriceFetcher {
         }
     }
 
+    pub async fn warmup(&self) -> Result<(), IngesterError> {
+        #[derive(serde::Deserialize)]
+        #[serde(rename_all = "camelCase")]
+        struct MintListItem {
```

Comment:
Why do we want a declaration of structs inside a function? Does it do something useful?

Reply:
the struct is single-use and used to deserialize only one particular response for one particular function, to me it looks logical & so much better than `.get("field")` used previously